testing/tests/DevExpress.ui.widgets.editors/colorBox.tests.js

```diff
@@ -578,6 +578,22 @@ QUnit.module('Color Box', {
         colorBox.open();
         assert.equal(colorBox._colorView.option('stylingMode'), 'underlined');
     });
+
+    [
+        { value: undefined, editAlphaChannel: false },
```

Comment:
Do we need check the '' value and 0 value? I guess that the result should be simular

Reply:
added '' value, digits have another behavior, there are displayed in input even without `editAlphaChannel`, but this logic is not within `_renderValue`